hw/ip/hmac/doc/checklist.md

```diff
@@ -54,6 +54,9 @@ Code Quality  | [LINT_PASS][]           | Done        |
 Code Quality  | [CDC_SETUP][]           | N/A         |
 Code Quality  | [FPGA_TIMING][]         | Done        | Fmax @ 50MHz on NexysVideo
 Code Quality  | [CDC_SYNCMACRO][]       | N/A         |
+Security      | [SEC_CM_IMPLEMENTED][]  | Not Started |
```

Comment:
@eunchan. Similar to comment on `AES` above, this plus other reasons makes me think HMAC should revert to `D1`. If you do revert, please convert all these to `Not Started`.

Reply:
Yeap. Revert and increase the version would be good.